plans/benchmarks/main.go

```diff
@@ -25,13 +25,8 @@ func run(runenv *runtime.RunEnv) error {
 		return BarrierBench(runenv)
 	case "subtree":
 		return SubtreeBench(runenv)
-	case "all":
```

Comment:
I am removing `all`, because `WaitNetworksInitialised` cannot be called twice right now, making this test to fail. Once we fix the sidecar, we might want to re-enable it again, but for now, we have to individually call different cases.